### PR TITLE
[Look&Feel] Use small context menu across the board

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description

Found 2 places that we need to use small context Menu. Along with #7057 , Now we have applied small context menu across the whole OSD codebase

image

## Changelog

- skip

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/home/public/application/components/sample_data_view_data_button.js [hunk 1]
```diff
@@ -71,18 +71,6 @@ export class SampleDataViewDataButton extends React.Component {
...
-        <EuiSmallButton
-          onClick={createAppNavigationHandler(dashboardPath)}
-          data-test-subj={`launchSampleDataSet${this.props.id}`}
-          aria-label={viewDataButtonAriaLabel}
-        >
-          {viewDataButtonLabel}
-        </EuiSmallButton>
-      );
-    }
-
```
Comment (virajsanghvi) [inline src/plugins/home/public/application/components/sample_data_view_data_button.js:85]: why did we remove a button?

**Review by virajsanghvi — COMMENTED** [review]
Rerunning ci to see if that addresses failures

File: src/plugins/home/public/application/components/sample_data_view_data_button.js (hunk 1 above)
Comment (zhongnansu) [reply src/plugins/home/public/application/components/sample_data_view_data_button.js]: it's by mistake, I add it back in the new commit

**Review by virajsanghvi — APPROVED** [review]
Change looks good - test failure looks unrelated so approving, rerunning just in case

**Review by zhyuanqi — APPROVED** [review]

**Pull request merged**